src/main/java/com/google/alpollo/model/Song.java

```diff
@@ -19,6 +21,15 @@ public Song(String artist, String name) {
     this.name = name;
   }
 
+  /** For objectify and saving maps. */
```

Comment:
can you provide more info, please? why do we need this method and how objectify use it